lib/ace/mode/latex_highlight_rules.js

```diff
@@ -19,7 +19,7 @@ var LatexHighlightRules = function() {
         }, {
             // A label
             token : ["keyword","lparen", "variable.parameter", "rparen"],
-            regex : "(\\\\(?:label|v?ref|cite(?:[^{]*)))(?:({)([^}]*)(}))?"
+            regex : "(\\\\(?:label|(?:v|eq)?ref|cite(?:[^{]*)))(?:({)([^}]*)(}))?"
```

Comment:
If my regex-foo is good, I take it that this matches `ref`, prefixed by either `v` or `eq`, which is good.

Are there any other variants we would want to match? If so, it might be worth making this something like `(?:[a-z]{1,2})?